src/style.css

```diff
@@ -0,0 +1,66 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 900;
+}
```

Comment:
why 2 `font-face`?

Reply:
I'm sorry, I didn't find how to merge 2 .ttf font files with different font weight in 1 @font-face? s it possible?

Reply:
No sorry, It's correct :)  well done!